src/plugins/discover/public/application/components/doc_viewer/doc_viewer.scss

```diff
@@ -1,82 +1,74 @@
-.kbnDocViewerTable {
-  @include euiBreakpoint('xs', 's','m') {
-    table-layout: fixed;
-  }
+@mixin docViewerCell {
+  align-items: flex-start;
+  padding: $euiSizeXS;
+  height: 100%;
 }
 
 .kbnDocViewer {
-  pre,
-  .kbnDocViewer__value {
-    display: inline-block;
-    word-break: break-all;
-    word-wrap: break-word;
-    white-space: pre-wrap;
-    color: $euiColorFullShade;
-    vertical-align: top;
-    padding-top: $euiSizeXS * .5;
-  }
-  .kbnDocViewer__field {
-    padding-top: $euiSizeS;
+  table,
+  tbody {
+    height: 100%;
```

Comment:
Setting `height: 100%` to (`table`, `tbody`, `tr`, `td`) enables to use 'align-items: flex-start' for cell content.